src/interface/Interface.py

```diff
@@ -0,0 +1,90 @@
+#Interface
+import sys
+from Tkinter import *
+from settings import SettingsGame
+
+
+class Interface(Frame):
+	
+	def __init__(self,main=None):
+
+		"""
+		main: tkinter frame object
+		contructor -- it will get Tkinter object to construct the main UI window.
+		this will call to initwindow() function , which will create and invoke all objects that will be avaiable in the mainwindow
+		
+		"""
+
+		Frame.__init__(self,main)
```

Comment:
An space after the coma. xD

*Note*: Is there some typos in the documentation. 
contructor line 12
avaiable line 13
zise line 25